sql/group.go

```diff
@@ -115,7 +120,7 @@ func (n *groupNode) Values() parser.DTuple {
 }
 
 func (n *groupNode) Next() bool {
-	if !n.needGroup {
+	if !n.needGroup || n.err != nil {
```

Comment:
I believe that is a convention we are using that you cannot call Next() once Next() returns an error.